redash/handlers/query_results.py

```diff
@@ -96,6 +96,15 @@ def run_query(query, parameters, data_source, query_id, max_age=0):
         return {'job': job.to_dict()}
 
 
+def get_download_filename(query_result, query=None, filetype='json'):
+    str_date = query_result.retrieved_at.strftime("%Y-%m-%d")
```

Comment:
could you rename this to `date` or `retrieved_at`? Same for `str_filename` (`filename`)